docs/pathways/ontology_curator.md

```diff
@@ -0,0 +1,44 @@
+# Ontology Curator Pathway
+
+## Getting Set-up
+
+1. Download and install [GitHub Desktop](https://desktop.github.com/)
+2. Download and install [Protege](https://protege.stanford.edu/products.php)
+3. Install [ELK reasoner in protege](../howto/installiong_elk_in_protege.md)
+4. [Setting up your ID range](../howto/idrange.md)
+5. [Setting up ODK](../howto/odk_setup.md)
+
+## Learning
+
+### Learning Git and GitHub
+
+1. [Fundamentals of GitHub](../tutorial/github_fundamentals.md)
+2. [Github issues](../tutorial/github_issues.md)
+3. [Cloning a Repo](../howto/clone_mondo_repo.md)
+4. [Creating pull requests](../howto/github_create_pull_request.md)
+
+### Learning how to edit ontologies
+1. [Contributing to an ontology](../lessons/contributing_to_obo_ontologies.md)
+2. [Avoid asserting redundant subclasses](../discussion/DeletingAssertedSubClasses.md)
+3. [When to use term comments](../discussion/TermComments.md)(Optional)
```

Comment:
I think we can add (Optional) in the pathways for things like Term Comments which can be a bit more divisive or stuff that is interesting discussion, but maybe a bit adjacent to a coherent pathway